compiler/build.gradle

```diff
@@ -10,6 +10,25 @@ repositories {
     mavenCentral()
 }
 
+task run(type: JavaExec) {
+    main = 'dk.aau.cs.d409f19.cellumata.MainKt'
+    classpath = sourceSets.main.runtimeClasspath
+
+    if (project.hasProperty("appArgs")) {
+        args Eval.me(appArgs)
+    }
+}
+
+jar {
+    manifest {
+        attributes 'Main-Class': 'dk.aau.cs.d409f19.cellumata.MainKt'
+    }
+
+    // This line of code recursively collects and copies all of a project's files
+    // and adds them to the JAR itself resulting in a 'fat jar'
+    from { configurations.compile.collect { it.isDirectory() ? it : zipTree(it) } }
```

Comment:
Doesn't gradle have builtin functions to handle the collections for classes?

Reply:
This is what stack overflow told me to do.